tests/test_client_device.py

```diff
@@ -805,31 +846,32 @@ def test_alert_reports(self):
         now = time.time()
         max_float_diff_1ms = (now + 0.001) / now - 1
 
-        for _activation_state, _actual_priority, _presence in product(list(pm_types.AlertActivation),
-                                                                      list(pm_types.AlertConditionPriority),
-                                                                      (True,
-                                                                       False)):  # test every possible combination
+        for _activation_state, _actual_priority, _presence in product(

```

Comment:
change import to `itertools.product`. check other usages

Reply:
I changed it to to import itertools